src/components/context/SearchContext.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { Dispatch, SetStateAction, createContext} from "react";
+
+// export const SearchContext = createContext({
```

Comment:
???

Reply:
I tried to use createContext but I accidentally pushed this